usaspending_api/database_scripts/etl/subaward_es_view.sql

```diff
@@ -2,106 +2,112 @@ DROP VIEW IF EXISTS subaward_es_view;
+	s.legal_entity_congressional_current AS recipient_location_congressional_code_current,
+	s.business_types,
+	s.place_of_perform_country_co AS pop_country_code,
+	s.place_of_perform_country_na AS pop_country_name,
+	s.place_of_perform_state_code AS pop_state_code,
+	s.place_of_perform_state_name AS pop_state_name,
+	s.place_of_performance_zip AS pop_zip,
+	s.place_of_perform_city_name AS pop_city_name,
+	s.place_of_perform_street AS pop_street,
+	s.place_of_perform_congressio AS pop_congressional_code,
+	s.place_of_performance_congressional_current AS pop_congressional_code_current,
+	s.award_description,
+	s.naics,
+	s.naics_description,
+	s.cfda_numbers as cfda_number,
+	s.cfda_titles,
+	s.subaward_type,
+	s.subaward_report_year,
+	s.subaward_report_month,
+	s.subaward_number,
+	s.subaward_amount,
+	s.sub_action_date,
+	s.sub_awardee_or_recipient_uniqu,
+	s.sub_awardee_or_recipient_uei,
+	s.sub_awardee_or_recipient_legal,
+	s.sub_dba_name,
+	s.sub_legal_entity_country_code AS sub_recipient_location_country_code,
+	s.sub_legal_entity_country_name AS sub_recipient_location_country_name,
+	s.sub_legal_entity_state_code AS sub_recipient_location_state_code,
+	s.sub_legal_entity_state_name AS sub_recipient_location_state_name,
+	s.sub_legal_entity_zip AS sub_recipient_location_zip,
+	s.sub_legal_entity_congressional AS sub_recipient_location_congressional_code,
+	s.sub_legal_entity_congressional_current AS sub_recipient_location_congressional_code_current,
+	s.sub_legal_entity_foreign_posta AS sub_recipient_location_foreign_posta,
+	s.sub_legal_entity_city_name AS sub_recipient_location_city_name,
+	s.sub_business_types,
+	s.sub_place_of_perform_country_co AS sub_pop_country_code,
+	s.sub_place_of_perform_country_name AS sub_pop_country_name,
+	s.sub_place_of_perform_state_code AS sub_pop_state_code,
+	s.sub_place_of_perform_state_name AS sub_pop_state_name,
+	s.sub_place_of_performance_zip AS sub_pop_zip,
+	s.sub_place_of_perform_congressio AS sub_pop_congressional_code,
+	s.sub_place_of_performance_congressional_current AS sub_pop_congressional_code_current,
+	s.sub_place_of_perform_city_name AS sub_pop_city_name,
+	s.sub_place_of_perform_street AS sub_pop_street,
+	s.subaward_description,
+	s.prime_award_group,
+	s.prime_award_type,
+	s.latest_transaction_id,
+	s.last_modified_date,
+	s.awarding_toptier_agency_name,
+	s.awarding_toptier_agency_abbreviation,
+	s.funding_toptier_agency_name,
+	s.funding_toptier_agency_abbreviation,
+	s.sub_fiscal_year,
+	s.business_categories,
+	s.type_of_contract_pricing,
+	s.type_set_aside,
+	s.extent_competed,
+	s.product_or_service_code,
+	s.product_or_service_description,
+    s.treasury_account_identifiers,
+    s.sub_ultimate_parent_unique_ide,
+    s.sub_ultimate_parent_uei,
+    a.disaster_emergency_fund_codes
 FROM
-	rpt.subaward_search
-WHERE action_date >= '2007-10-01';
\ No newline at end of file
+	rpt.subaward_search s
+LEFT JOIN rpt.award_search a
```

Comment:
Shoot! This creates a new dependency, meaning that the Subaward ES indexer can't run until the Award Search Delta Table finishes refreshing. Definitely not an easy way around it though. I think we have two options:
1. Leave this JOIN here and move the Subaward Reindex step back in the pipeline to after the Award Search Delta Table refresh
2. Calculate the DEFCs in the Subaward Search Delta Table refresh. `int.awards` is already being joined to, but that table doesn't have DEFC information. An extra join would have to be added to `financial_accounts_by_awards`

I think option 2 would be the better long term solution, but it would lead to significant scope creep. I'm okay with sticking to option 1 if you can create a ticket for option 2 that we can implement if we need to based on pipeline runtimes.

Reply:
I like option 1